main.code-workspace

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
What's this?

Reply:
O that is the IDE config file from my editor I thought I added it to the .gitignore. Must have forgotten to do that. Sorry for that.

Reply:
Hah doh!